incubator/hnc/pkg/controllers/hierarchy_controller.go

```diff
@@ -262,10 +263,27 @@ func (r *HierarchyReconciler) syncLabel(log logr.Logger, nsInst *corev1.Namespac
 		return
 	}
 
+	// pre-define HNC label suffix
+	labelSuffix := fmt.Sprintf(".tree.%s/depth", api.MetaGroup)
+
+	// Remove all existing HNC labels.
+	for k := range nsInst.Labels {
+		if strings.HasSuffix(k, labelSuffix) {
+			delete(nsInst.Labels, k)
+		}
+	}
+
+	// Remove all existing HNC annotations.
```

Comment:
I know I said "annotations" in the issue, but this is the only function that puts anything on the namespace and there don't appear to be any annotations (yet). If we do add any, they won't end in `/depth`. So we should remove this bit.

Reply:
removed